package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.6.3",
+  "version": "6.6.4",
```

Comment:
Please double-check the version in `master` before merging, and update accordingly? There are several open PRs right now. Thank you!!